docarray/computation/torch_backend.py

```diff
@@ -73,6 +73,50 @@ def none_value() -> Any:
         """Provide a compatible value that represents None in torch."""
         return torch.tensor(float('nan'))
 
+    @staticmethod
+    def shape(tensor: 'torch.Tensor') -> Tuple[int, ...]:
+        return tuple(tensor.shape)
+
+    @overload
+    @staticmethod
+    def reshape(tensor: 'TorchTensor', shape: Tuple[int, ...]) -> 'TorchTensor':
+        """
+        Gives a new shape to tensor without changing its data.
+
+        :param tensor: tensor to be reshaped
+        :param shape: the new shape
+        :return: a tensor with the same data and number of elements as tensor
+            but with the specified shape.
+        """
+        ...
+
+    @overload
```

Comment:
Would be nice to automatically generate the overload. But lets not do it in this PR unless you found a brillant solution haha

Reply:
I dont think it works :( https://github.com/python/mypy/issues/11488

Reply:
but we could generate the code though

Reply:
but yeah lets see later

Reply:
nah not code generation via our friend the bot.

But we could have a helper function that generate 3 methods with the correct overload

Reply:
could it be done via decorator somehow?